src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -277,12 +277,17 @@ def list_partitions(self,
             for partition_prefix in prefix.partition_prefixes()
         ]
         assert prefixes, prefix
-        entity_type = BundleEntityType.primary.value
-        pk_column = entity_type + '_id'
+        primary = BundleEntityType.primary.value
+        supplementary = BundleEntityType.supplementary.value
         rows = self._run_sql(f'''
-            SELECT prefix, COUNT({pk_column}) AS subgraph_count
-            FROM {backtick(self._full_table_name(source.spec, entity_type))}
-            JOIN UNNEST({prefixes}) AS prefix ON STARTS_WITH({pk_column}, prefix)
+            SELECT prefix, COUNT(datarepo_row_id) AS subgraph_count
```

Comment:
Could you also explain why using the PK instead of datarepo_row_id is a bug? How did it manifest itself?

Reply:
Resolved at PL.